InteractiveHtmlBom/ecad/kicad.py

```diff
@@ -322,8 +322,12 @@ def parse_drawing(self, d):
         elif d.GetClass() in ["PTEXT", "MTEXT", "FP_TEXT", "PCB_TEXT"]:
             s = self.parse_text(d)
         elif d.GetClass().startswith("PCB_DIM"):
```

Comment:
add this condition instead:
`and hasattr(pcbnew, "VECTOR_SHAPEPTR")`

Reply:
Ok, I changed the patch like this